src/app/modules/consultations/consultations-summary/consultations-summary.component.ts

```diff
@@ -91,11 +99,32 @@ export class ConsultationsSummaryComponent implements OnInit {
     if (responsesList.length) {
       this.annonymousResponses = responsesList.filter(response => (response.node.user === null || response.node.user === undefined));
       this.publicResponses = responsesList.filter(response => response.node.user !== null);
+      this.publicResponsesLength = this.publicResponses.filter((response) => response.node.roundNumber === this.roundNumber).length;
+      this.annonymousResponsesLength = this.annonymousResponses.filter((response) => response.node.roundNumber === this.roundNumber).length;
       this.summaryData = {
         publicResponseCount : this.publicResponses.length,
         title: 'RESPONSES TO'
       };
     }
   }
 
+  getActiveRound(profileData) {
+    if (profileData && profileData.responseRounds) {
+      const responseRounds = profileData.responseRounds;
```

Comment:
@nishanthh Again, I don't think you need to define a function variable for responseRounds as there is already defined in line 84